src/Symfony/Component/Config/Tests/Resource/GlobResourceTest.php

```diff
@@ -47,6 +47,32 @@ public function testIterator()
         $this->assertSame($dir, $resource->getPrefix());
     }
 
+    public function testIteratorForExclusionDoesntIterateThroughSubfolders()
+    {
+        $dir = \dirname(__DIR__).\DIRECTORY_SEPARATOR.'Fixtures';
+        $resource = new GlobResource($dir, '/Exclude', true, true);
+
+        $paths = iterator_to_array($resource);
+
+        $file = $dir.\DIRECTORY_SEPARATOR.'Exclude';
+        $this->assertArrayHasKey($file, $paths);
+        $this->assertCount(1, $paths);
+    }
+
+    public function testIteratorSkipsFoldersForGivenExcludedPrefixes()
+    {
+        $dir = \dirname(__DIR__).\DIRECTORY_SEPARATOR.'Fixtures';
+        $resource = new GlobResource($dir, '/*Exclude*', true, false, array($dir.\DIRECTORY_SEPARATOR.'Exclude' => true));
```

Comment:
please also add a test using `/` in the exclude path rather than `\DIRECTORY_SEPARATOR`, to ensure it works

Reply:
Done.